fe/fe-core/src/main/java/org/apache/doris/analysis/InsertStmt.java

```diff
@@ -458,8 +458,20 @@ private void analyzeSubquery(Analyzer analyzer) throws UserException {
             if (column.isNameWithPrefix(CreateMaterializedViewStmt.MATERIALIZED_VIEW_NAME_PREFIX)) {
                 SlotRef refColumn = column.getRefColumn();
```

Comment:
maybe mv_ is a stop words in doris. So I think we should throw excetion when user create column starts with mv_

Reply:
ok, i add some check before create table or column